### PR TITLE
Document::download encoding fix

### PR DESCRIPTION
**Pull request by 0xSekar**
Fix for issue #231

### CODE REVIEW

File: lib/Document.php [hunk 1]
```diff
@@ -106,10 +108,14 @@ public function download(?bool $postProcess = true): string {
             return $contents;
         }
 
-        // Documents are ISO-8859-1 encoded, which messes up the data when we read it directly
+        // Documents encodings depend on target marketplace, which messes up the data when we read it directly
```
Comment (jlevers) [inline lib/Document.php:111]: Please update this to say:

```php
// Document encodings depend on the target marketplace. English-language reports are
// typically ISO-8859-1 encoded, which messes up the data when we read it directly via
// SimpleXML or as a plain TAB/CSV, but the original encoding is required to parse XLSX
// and PDF reports.
```

**Review by jlevers — CHANGES_REQUESTED** [review]
Looks good -- just want an update on the relevant comment.

File: lib/Document.php (hunk 1 above)
Comment (0xSekar) [reply lib/Document.php]: Updated

**Review by jlevers — APPROVED** [review]
Looks great, thanks!

**Pull request merged**